source/options.css

```diff
@@ -2,12 +2,24 @@ body {
 	min-width: 400px; /* Required to size the Browser Action popup */
 }
 textarea {
+	display: block;
 	width: 100%;
+	font-family: monospace !important;
+	font-size: 12px;
+	line-height: 1.5;
+	resize: vertical;
+}
+label {
+	vertical-align: -0.16em;
```

Comment:
@bfred-it Can you verify this value of `-0.16em`? Gives me `2px` for `12px` font size (`2 / 12 ~= 0.16`).